api/cl_ext_float_atomics.asciidoc

```diff
@@ -0,0 +1,20 @@
+// Copyright 2018-2024 The Khronos Group Inc.
+// SPDX-License-Identifier: CC-BY-4.0
+
+include::{generated}/meta/{refprefix}cl_ext_float_atomics.txt[]
+
+=== Other Extension Metadata
+
+*Last Modified Date*::
+    2020-08-12
+*IP Status*::
+    No known IP claims.
+
+=== Description
+
+This extension is described in the OpenCL Extensions document.
```

Comment:
Agree but it's hard to provide a link that's valid regardless of the spec you're looking at. Links would be different for published PDF or HTML versions of the spec and different again for the dev versions. I felt this was a bit too hard for a "temporary" (tm) problem.

Reply:
cool, we can leave this out of it isn't trivial.

Reply:
Note, the EXT extensions actually aren't currently in the OpenCL Extensions document, and they're documented in standalone extension docs on the registry, similar to vendor extensions.  For example:

https://registry.khronos.org/OpenCL/extensions/ext/cl_ext_float_atomics.html

Until this changes, maybe it is possible to put a stable link to the appropriate extension document?

Reply:
I was looking at the PDF when writing this. The spec tooling generates:
- `out/pdf/OpenCL_Ext.pdf` titled "The OpenCL Extension Specification"
- `out/pdf/extensions.pdf` titled "OpenCL Extensions"

It's the latter that this refers to and that one does include EXT extensions. I agree there is potential for confusion and the real fix IMO is to get rid of both these documents :).

We could do what you propose but this would always link to the *latest pulished* version of the specification in HTML form. It would likely be useful to a good number of users but would at the same time potentially point to the wrong document for some others.

Reply:
The tooling generates "extensions.pdf", but AFAIK we don't publish it.  We still do publish "OpenCL_Ext.pdf", but we might not do so for long - see #1111.

What's the disadvantage of linking to the HTML spec?  The way I look at it, if we don't put any link, readers looking for more information about the extension are either going to get frustrated and give up, or they're going to search for the name of the extension, which will (hopefully!) point them to the HTML spec on the registry.  If that's the case, why not give them the link directly?

Reply:
Beyond having PDFs link to the HTML specs, the main issue I have with adding these links is that the version linked would not be the one aligned to the document linking to it. It's not an issue for published versions of the specification but if I package a set of PDFs generated from an arbitrary SHA for example, the link would potentially point to the wrong spec text. This mostly affects IHVs so maybe not a good reason to deprive OpenCL users of a helpful link even if only temporary so. I'll add the links with text stating that they point to the "latest published" version of the specifications.